src/pages/Account.vue

```diff
@@ -293,13 +308,21 @@ export default {
                 : "";
         },
         pgChangePage: function (event) {
+            this.loadSpinner();
             const key = event.target.id;
             const pg = this.voucherPagination || {};
             const page = pg.hasOwnProperty(key) ? pg[key].page : 1;
             Store.getVoucherPaymentState(page);
         },
+        loadSpinner: function () {
+            this.showSpinner();
```

Comment:
Currnently this looks like
- mount the page
- show the spinner
- set a timeout to hide th spinner at 2 aeconds
- fetch the data <- may take more than 2 seconds
- time out triggers
- data may not have arrived.

It's OK for now, but we should remember to tie spinner hiding to data-has-been-fetched when we fix that bit.

And I don't have a concrete idea about that at the moment.
I'm thinking
- Store has a flag that says it's fetching for us
- we watch that in data here
- when it changes to true, we start a timer to show spinner
- when it goes false, we cancel timer and hide spinner. 
- if it takes longer than timer, spinner shows
But that's for another card.